### ISSUE TITLE
Update ARIA roles and labels

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#86]
--- 
- Remove `role="alert"` on all the alert-styled divs. This role is only for time-sensitive information that the screen reader should announce immediately. These in-line alert-style divs don't need to be announced immediately. [ARIA: alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/alert_role).
<img width="840" height="170" alt="Image" src="https://github.com/user-attachments/assets/e566a8ed-169c-4ee2-8745-48b467a058a3" />
<img width="849" height="72" alt="Image" src="https://github.com/user-attachments/assets/567e4254-28d5-46cc-8b84-9122f40ab7a5" />

---
- Remove `role="banner"` on the Welcome banner message. This role is equivalent to the page `<header>` and not appropriate in this context. Instead, add `role="region" aria-label="Welcome"` to the Welcome banner message. [ARIA: banner role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/banner_role).
<img width="907" height="227" alt="Image" src="https://github.com/user-attachments/assets/c27c7516-1d08-4b2e-9624-803667fd1dc0" />

---
- Change the Progress bar `<div>` to an `<aside>` and add `aria-label="Progress"`.

---
- For each step (1-7) in the submission: change the outermost `<div>` (i.e., the one with `class="card"`) to a `<section>` and add `aria-labelledby="[step# id] [heading id] [step status id]"` where:
  - [step# id] uniquely references the div containing the step's number.
  - [heading id] uniquely references the step's `<h2>`.
  - [step status id] uniquely references the step's progress badge.

[WCAG: Using aria-labelledby to concatenate a label from several text nodes](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA9).

Note: This section label should update corresponding to the step's status (In Progress or Completed). To do this, it may require wrapping the badge divs in a parent div and referencing that. A different, possibly simpler approach may be to wrap all three (step#, heading, and status) in the `<h2>` and reference only that element.

---
